symplyphysics/laws/thermodynamics/equation_of_heat_balance.py

```diff
@@ -0,0 +1,32 @@
+from sympy import (Eq, solve)
```

Comment:
BTW `heat_balance` is not very descriptive. What if we rename law to `sum_of_heat_transfer_is_zero`?
Also it looks like a conservation law. Maybe move it to `conservation` category?

Reply:
I also thought that this is a kind of law of conservation of energy, more precisely, as a kind of differential form of it, since here energy is transferred during heating/cooling between parts of the system. Heating/cooling lasts for some time, which means that the change in total energy at any given time is zero, that is, similar to the conservation of total energy: $E(t_1) - E(t_2) = 0$.

There is a mathematically similar formula in electricity: Kirchhoff's second law. But there the sum of the currents in the node is zero. By analogy with him, he created this law.